lib/template/templates.c

```diff
@@ -395,6 +396,7 @@ log_template_options_defaults(LogTemplateOptions *options)
   options->frac_digits = -1;
   options->ts_format = -1;
   options->on_error = -1;
+  options->use_fqdn = TRUE;
```

Comment:
GlobalConfig's `use_fqdn` option is FALSE by default. Is this default okay then?

Reply:
good point. I've changed this to FALSE and also added tests to see if LOGHOST honours the setting.

I was thinking about an "upgrade" feature, but then decided against it, as it would spread GlobalConfig dependency to LogMessage, which I felt was not warranted.

I can still add a warning on the change for older config versions, but since I think this is actually a bugfix, I haven't done it. Let me know if you disagree.

Reply:
btw, documentation should probably need to be clarified, it is very short on this macro, saying it matches the output of the hostname call. @fekete-robert